app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -0,0 +1,51 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/train-to-be-a-teacher/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
+
+            <p>
+                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options depending on your situation.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-have-experience" class="category__nav-card">
+            <h2>If you have classroom experience</h2>
+
+            <p>
+                If you have already spent some time working in the classroom, you may be able to follow the assessment only route to show you meet QTS.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-want-to-change-career" class="category__nav-card">
+            <h2>If you want to change career</h2>
+
+            <p>
+            Use the experience you’ve gathered so far to inspire students and bring your skills and expertise to the classroom.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/other-training-you-can-get" class="category__nav-card">
+            <h2>Other training and experience you can get</h2>
+
+            <p>
+                Whatever your situation there’s a training programme for you, from spending some time at a school to internships.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/what-subject-do-you-want-to-teach" class="category__nav-card">
+            <h2>What subject do you want to teach?</h2>
+
+            <p>
+                If you want to teach a particular subject there may be extra funding available or subject specific training.
```

Comment:
This feels like it could lead to a negative experience for some people who want to teach a subject without funding or training!

When we talk about "subject-specific training", are you meaning things like Engineers teach physics (niche!) or about enhancing your knowledge (SKEs - https://getintoteaching.education.gov.uk/improve-your-subject-knowledge)? I wonder if SKEs needs a card/page all to itself as it is important?